src/sentry/static/sentry/app/components/gridEditable/index.tsx

```diff
@@ -146,13 +145,18 @@ class GridEditable<
     isEditing: false,
   };
 
+  componentDidMount() {
+    window.addEventListener('resize', this.redrawGridColumn);
```

Comment:
Should this be throttled or debounced?

Reply:
I tested debounced but it didn't have any performance improvements, so I kept this simple approach. 

Resizing the window on other websites like Google/Github causes some skipped frames too, so it might be a OS/browser problem.